kubeflow/core/prototypes/all.jsonnet

```diff
@@ -6,9 +6,11 @@
 // @optionalParam namespace string default Namespace
 // @optionalParam disks string null Comma separated list of Google persistent disks to attach to jupyter environments.
 // @optionalParam cloud string null String identifying the cloud to customize the deployment for.
+// @optionalParam tfAmbassadorServiceType string ClusterIP The service type for the API Gateway.
 // @optionalParam tfJobImage string gcr.io/kubeflow-images-staging/tf_operator:v20180226-403 The image for the TfJob controller.
 // @optionalParam tfDefaultImage string null The default image to use for TensorFlow.
 // @optionalParam tfJobUiServiceType string ClusterIP The service type for the UI.
+// @optionalParam tfDefaultTensorboardImage string gcr.io/google-samples/appengine-tensorboard:latest The service type for the UI.
```

Comment:
Description for `tfDefaultTensorboardImage ` is incorrect

Reply:
thanks @ankushagarwal - i'll break this into 2 PRs, correct tfDefaultTensorboardImage description and provide tests for util.toArray